tests/test_queue_model.py

```diff
@@ -291,7 +289,7 @@ def make_script_info(self, is_standard=False, path=None, config="wait_time: 0.1"
 
     def next_visit_callback(self, script_info):
         dt = time.monotonic() - self.t0
-        print(
+        self.log.debug(
```

Comment:
Why?

Reply:
Mixing print's with log messages generates really confusing reports when running pytest.

Reply:
I'm having trouble figuring out why it matters, but I I guess I should try it and see what the difference is.

Reply:
Well, I think it all boils down to “why” the print statement is there. I imagine it is for debugging purposes. If that is the case, using the log facility alone should be a no brainer as it gives way more control to the users on how to display and store that. For instance, you can log that to a file using pytest facility. It is also possible to set out filters and such. 

If the prints are for some other reason, I imagine an alternative would be to remove them altogether.